### ISSUE TITLE
Support for applying Plugin to Services

### ISSUE THREAD

**Issue by KenFigueiredo** [kevholditch/terraform-provider-kong#44]
Hey there,
Doesn't look like there is a method to apply plugins to a given service id.
E.g : https://docs.konghq.com/0.14.x/getting-started/enabling-plugins/

Is this currently WIP? If not I could try to make a PR for it

**Comment by kevholditch** [comment]
@KenFigueiredo no one is currently working on this if you want to create a PR, thanks

**Comment by quantumew** [comment]
I am assuming this is still an outstanding feature. Going to open a PR for this one this week unless someone is on it already.

**Comment by kevholditch** [comment]
Fixed now in the provider version `v2.0.0` onwards.  Note that provider only supports kong `v1.0.0` or greater.  Simply set `service_id` on the `plugin` resource to the `service_id` for example:

```hcl
resource "kong_service" "service" {
	name     = "test"
	protocol = "http"
	host     = "test.org"
}

resource "kong_plugin" "rate_limit" {
	name        = "rate-limiting"
	service_id = "${kong_service.service.id}"
	config_json = <<EOT
	{
		"second": 10,
		"hour" : 2000
	}
EOT
}
```
